po/de.po

```diff
@@ -2276,12 +2276,18 @@ msgid ""
 "Entry column widths in Invoices, Bills, and Employee Vouchers may now be "
 "saved as defaults for each type of document."
 msgstr ""
+"Die Breite der Eingabespalte in Lieferatenrechnungen, Rechnungen und "
+"Mitarbeiterdokumente können jetzt als Standardwerte für jeden Dokumenttyp "
```

Comment:
Sinngemäß wie oben.

Reply:
@fellen: habe ich in Weblate korrigiert.